### ISSUE TITLE
Cannot find module '.../@nrwl/expo/./src/migrations/updae-14-1-2/add-eject-target-14-1-2.ts'

### ISSUE THREAD

**Issue by makhataibar** [nrwl/nx-labs#72]
<img width="1728" alt="Снимок экрана 2022-06-13 в 11 21 21" src="https://user-images.githubusercontent.com/27006656/173285215-5363225e-5fde-401a-81b6-3ef12654d27c.png">

Can't update @nrwl/expo to latest
**Steps:**

- npx nx migrate @nrwl/expo
- npx nx migrate --run-migrations

**Comment by makhataibar** [comment]
I see it's fixed in commit https://github.com/nrwl/nx-labs/commit/30e602dcc15754b3650f87a0892bf31c1646904b,
but it doesn't have in latest @nrwl/expo package

**Comment by xiongemi** [comment]
it should be fixed for version 14.3.0.
the migration would be:
```
{
  "migrations": [
    {
      "version": "14.1.2-beta.0",
      "cli": "nx",
      "description": "Add target eject for expo projects in project.json",
      "factory": "./src/migrations/update-14-1-2/add-eject-target-14-1-2",
      "package": "@nrwl/expo",
      "name": "add-eject-target-14-1-2"
    }
  ]
}
```
